sdk/tables/data-tables/test/testUtils.ts

```diff
@@ -9,3 +9,5 @@ export const isNode =
   !!process.version &&
   !!process.versions &&
   !!process.versions.node;
+
+export const isNode8 = (): boolean => isNode && Number(process.versions.node.split(".")[0]) === 8;
```

Comment:
should we have this in the recorder or other test util package instead?

Reply:
Sounds fair!
Moving to the brand new `@azure/test-utils`.